test/test_pool.py

```diff
@@ -126,17 +127,17 @@ async def worker():
     ), f"{nworkers} workers are waiting for a connection"
     ev.set()  # allow the workers to get their connections
     await counter2.wait()
-    assert (
-        pool.currently_allocating == 0 and pool.in_use == nworkers
-    ), "all workers should have their connections now"
+    assert pool.currently_allocating == 0
+    assert pool.in_use == nworkers, "all workers should have their connections now"
     ev2.set()
     await asyncio.gather(*coros)
+    assert pool.currently_allocating == 0
     assert (
-        pool.in_use == 0 and pool.available.qsize() == nworkers
```

Comment:
Was the `pool.in_use == 0` assertion removed intentionally?

Reply:
Oversight - nice catch